oauth2_http/javatests/com/google/auth/oauth2/DefaultCredentialsProviderTest.java

```diff
@@ -326,6 +336,48 @@ public void getDefaultCredentials_envUser_providesToken() throws IOException {
     testUserProvidesToken(testProvider, USER_CLIENT_ID, USER_CLIENT_SECRET, REFRESH_TOKEN);
   }
 
+  @Test
+  public void getDefaultCredentials_envGdchServiceAccount_providesToken() throws IOException {
```

Comment:
add test cases for cert validation

Reply:
make sure that if credential has valid path - it creates a transport with it, and fail otherwise

Reply:
we have cert validation tests in GdchCredentialsTest